ci-operator/step-registry/telcov10n/metal-single-node-spoke-kpis/tests/cpu-util/telcov10n-metal-single-node-spoke-kpis-tests-cpu-util-commands.sh

```diff
@@ -0,0 +1,253 @@
+        ansible.builtin.shell: |
+          cat << EO-Containerfile > {{ _remote_container_file }}
+          FROM quay.io/centos/centos:stream9
+          RUN dnf -y install binutils diffutils skopeo git python3 python3-pip python3-devel jq && \
+            dnf clean all && \
+            curl -sSLO {{ _oc_bin_url }} && \
+            tar -zxvf {{ _oc_bin_url | basename }} && \
+            mv oc /usr/local/bin/ && \
+            mv kubectl /usr/local/bin/ && \
+            rm -f {{ _oc_bin_url | basename }}
+          ENV TELCO_KPI_TEST_NAME=${TELCO_KPI_TEST_NAME}
+          EO-Containerfile
+
+          podman build \
+            -t {{ _helper_image_tag }} \
+            -f Containerfile \
+            {{ _remote_container_file | dirname }}
+
+      - name: Running the ${TELCO_KPI_TEST_NAME} test
+        ansible.builtin.shell: |
+          mkdir -pv {{ _remote_telco_ran_integration_repo }}
+          podman run --rm -ti \
+              \
+              -e KUBECONFIG={{ _remote_kubeconfig }} \
+              -e SCRIPTS_DIR="{{ _remote_telco_ran_integration_repo }}/scripts" \
+              -e DURATION="${TELCO_KPI_TEST_DURATION}" \
+              \
+              -v {{ _remote_kubeconfig }}:{{ _remote_kubeconfig }}:Z \
+              -v {{ _remote_telco_ran_integration_repo }}:{{ _remote_telco_ran_integration_repo }}:Z \
+              -v {{ _remote_run_cmd }}:{{ _remote_run_cmd }}:Z \
+              \
+              {{ _helper_image_tag }} bash {{ _remote_run_cmd }}
+
+      - name: Gather ${TELCO_KPI_TEST_NAME} tests logs and results
+        ansible.builtin.shell: |
+          find {{ _remote_telco_ran_integration_tests_results }} -maxdepth 1 -type f
+        register: _telco_kpi_artifacts
+
+      - name: Fetch results and store them into SHARED folder
+        ansible.builtin.fetch:
+          src: "{{ item }}"
+          dest: "{{ _test_results_artifacts_append }}_{{ item | basename }}"
+          flat: true
+        loop: "{{ _telco_kpi_artifacts.stdout_lines }}"
+
+    always:
+
+      - name: Remove the remote temporary dir
+        when: _telco_kpis.path is defined
+        ansible.builtin.file:
+          path: "{{ _telco_kpis.path }}"
+          state: absent
+EO-playbook
+
+  echo
+  echo "----------------------------------------"
+  ls -l ${ansible_playbook}
+  echo "----------------------------------------"
+  cat ${ansible_playbook}
+  echo "----------------------------------------"
+  echo
+}
+
+function run_ansible_playbook {
+
+  echo
+  echo "Running KPIs tests..."
+  echo
+
+  set -x
+  oc get no,clusterversion -owide
+  set +x
+
+  ansible-playbook -i ${inventory_file} ${ansible_playbook} -vvv
+}
+
+function setup_test_result_for_component_readiness {
+
+  set -x
+  sed -E \
+    -e 's/(<testsuite name=")[^"]+/\1telco-verification/' \
+    -e "s/<testcase name=\"([^\"]+)\"/<testcase name='${TEST_COMPONENT} \1'/" \
+    ${test_results_artifacts_append}*.xml \
```

Comment:
Here it is the problematic point. It would reapply the same transformation done to previos KPI tests. Currently it will work due to `TEST_COMPONENT` has the same value in both cases (oslat and cpu-utils). But that may change in the future and would lead to a _hide_ issue.

Current:
`test_results_artifacts_append=${SHARED_DIR}/telco_kpi_results`

Proposed change:
`test_results_artifacts_append=${SHARED_DIR}/telco_${TELCO_KPI_TEST_NAME}_kpi_results`